src/git.rs

```diff
@@ -1,24 +1,25 @@
 use chrono::{TimeZone, Utc};
-use git2::{Error, ErrorCode, Repository};
+use git_repository as git;
 
 thread_local! {
-    static REPOSITORY: Result<Repository, Error> = Repository::open(".");
+    static REPOSITORY: Result<git::Repository, git::discover::Error> = git::discover(".");
 }
 
 byond_fn!(fn rg_git_revparse(rev) {
-    REPOSITORY.with(|repo| -> Result<String, ErrorCode> {
-        let repo = repo.as_ref().map_err(Error::code)?;
-        let object = repo.revparse_single(rev).map_err(|e| e.code())?;
-        Ok(object.id().to_string())
+    REPOSITORY.with(|rep| -> Result<String, String> {
```

Comment:
Why did you make this variable less readable?

Reply:
I thought it was clearer that this was the argument, because there is a different local variable (not a shadowed one or anything) named `repo` within the actual block. I can change it back if you want.

Reply:
Shadowing names in let redefs like this is pretty common in rust code